content-src/experiments/notes.yaml

```diff
@@ -63,7 +64,7 @@ tour_steps:
     image: /static/images/experiments/notes/notes-tour-5.jpg
     copy: You can always give us feedback or disable Notes from Test Pilot.
   -
-    image: /static/images/experiments/notes/notes-tour-5.jpg
+    image: /static/images/experiments/notes/notes-tour-6.jpg
```

Comment:
Does this mean we just orphaned the "notes-tour-5.jpg" file and should just delete it from the repo?


![](https://raw.githubusercontent.com/mozilla/testpilot/master/frontend/src/images/experiments/notes/notes-tour-5.jpg)

**Figure 1:** [/frontend/src/images/experiments/notes/notes-tour-5.jpg](https://github.com/mozilla/testpilot/blob/master/frontend/src/images/experiments/notes/notes-tour-5.jpg)